From 922f95b2700a03bb312845f4bcae19ffbd7118a3 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Wed, 15 Aug 2018 20:14:20 +0800
Subject: [PATCH 09/29] arm64: allwinner: dts: h5: fix the VDD-CPUX voltage of
 ALL-H3-CC H5 ver

The VDD-CPUX voltage of the H5 ver of Libre Computer ALL-H3-CC is not
1.2V but 1.1V.

Fix this in the device tree.

Fixes: af5d05bdc99c("arm64: dts: allwinner: Add dts file for Libre Computer Board ALL-H3-CC H5 ver.")
Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 .../boot/dts/allwinner/sun50i-h5-libretech-all-h3-cc.dts     | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h5-libretech-all-h3-cc.dts b/arch/arm64/boot/dts/allwinner/sun50i-h5-libretech-all-h3-cc.dts
index d811df332824..31206a9eab4b 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h5-libretech-all-h3-cc.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h5-libretech-all-h3-cc.dts
@@ -15,3 +15,8 @@ / {
 &mmc2 {
 	mmc-ddr-3_3v;
 };
+
+&reg_vdd_cpux {
+	regulator-min-microvolt = <1100000>;
+	regulator-max-microvolt = <1100000>;
+};
-- 
2.30.2

